src/main/resources/testTicketsP1.json

```diff
@@ -0,0 +1,12 @@
+{
+  "objects": [
```

Comment:
If these are integration tests we should connect to real system, not just mock it.
That's what we do in most plugins.

Example:
https://github.com/data-integrations/salesforce/blob/9ec60b8ddc9dce94303a0021a3ba305b7cf1063a/src/test/java/io/cdap/plugin/salesforce/etl/BaseSalesforceETLTest.java#L67

If that's not possible please specify why.

Reply:
Usually the flow is:
1) Connect to real system specified credentials etc are specified in -D params.
2) Create some data in the system via tests.
3) Run tests on it.
4) Delete data.